docs/pages/database-access/guides/elastic.mdx

```diff
@@ -0,0 +1,248 @@
+  "metadata" : {
+    "version" : 1
+  }
+}
+'
+```
+
+## Step 4/5. Set up mutual TLS
+
+(!docs/pages/includes/database-access/tctl-auth-sign.mdx!)
+
+We will show you how to use `tctl auth sign` below.
+
+To securely access your Elasticsearch database, sign the certificate for the hostname Teleport will
+connect to:
+
+For example, if your Elasticsearch server is accessible at `elastic.example.com`,
+run:
+
+```code
+$ tctl auth sign --format=elastic --host=elastic.example.com --out=elastic --ttl=2160h
+Database credentials have been written to elastic.key, elastic.crt, elastic.cas.
+```
+
+(!docs/pages/includes/database-access/ttl-note.mdx!)
+
+The command will create three files:
+
+- `elastic.cas` with Teleport's certificate authority
+- `elastic.key` with a generated private key
+- `elastic.crt` with a generated user certificate
+
+You will need these files to enable mutual TLS on your Elasticsearch server.
+
+(!docs/pages/includes/database-access/rotation-note.mdx!)
+
+Use the generated secrets to enable mutual TLS in your `elasticsearch.yml` configuration
+file:
+
+```yml
+xpack.security.http.ssl:
+  certificate_authorities: /path/to/elastic.cas
+  certificate: /path/to/elastic.crt
+  key: /path/to/elastic.key
+  enabled: true
+  client_authentication: required
+  verification_mode: certificate
+
+xpack.security.authc.realms.pki.pki1:
+  order: 1
+  enabled: true
+  certificate_authorities: /path/to/elastic.cas
+```
+
+<Admonition type="tip">
+For more information on configuring security settings in Elasticsearch, see:
+[Security settings in Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-settings.html)
+</Admonition>
+
+Once mutual TLS has been enabled, you will no longer be able to connect to the cluster without
+providing a valid client certificate. You can set `xpack.security.http.ssl.client_authentication`
+to `optional` to allow connections from clients that do not present a certificate.
+
+## Step 5/5. Connect
+
+Log into your Teleport cluster and see available databases:
+
+<Tabs>
+  <TabItem label="Self-Hosted">
+    ```code
+    $ tsh login --proxy=teleport.example.com --user=alice
+    $ tsh db ls
+    Name                        Description Allowed Users Labels  Connect
+    --------------------------- ----------- ------------- ------- ------------------------
+    > myelastic (user: elastic)             [*]           env=dev tsh db connect myelastic
+    ```
+  </TabItem>
+  <TabItem label="Teleport Cloud">
+    ```code
+    $ tsh login --proxy=mytennant.teleport.sh --user=alice
+    $ tsh db ls
+    Name                        Description Allowed Users Labels  Connect
+    --------------------------- ----------- ------------- ------- ------------------------
+    > myelastic (user: elastic)             [*]           env=dev tsh db connect myelastic
+    ```
+  </TabItem>
+</Tabs>
+
+To connect to a particular database instance:
+
+```code
+$ tsh tb connect myelastic --db-user=elastic
```

Comment:
```suggestion
$ tsh db connect myelastic --db-user=alice
```

Reply:
I fixed the `tb` typo, but the question about `--db-user` remains.

Reply:
@alexfornuto Apologies, I missed your comment. `elastic` is a special built-in user and unlikely to be used by regular end users. We also added a mapping granting `alice` `superuser` role. The username which Elasticsearch sees - here `alice` - is whatever is provided to `--db-user`. The Teleport user must be allowed to use that `--db-user` and we ensure that with appropriate traits/roles combination.